core/plugins/org.polarsys.capella.core.ui.search/src/org/polarsys/capella/core/ui/search/result/CapellaSearchResultPage.java

```diff
@@ -63,4 +68,17 @@ public void setSelection(ISelection selection) {
   public ISelection getSelection() {
     return getViewer().getSelection();
   }
+
+  protected SearchResultContentProvider getViewerContentProvider() {
+    return (SearchResultContentProvider) getViewer().getContentProvider();
+  }
+
+  /**
+   * Make the viewer public.
+   */
+  @Override
+  public StructuredViewer getViewer() {
+    return super.getViewer();
+  }
+

```

Comment:
This method is made public (protected in the mother class), to facilitate the coupling between **CapellaSearchPage** and each of its 2 **ContentProviders**.